### ISSUE TITLE
Windows Taskbar Jump List repeats same labels because of workspace file

### ISSUE THREAD

**Issue by sleutho** [microsoft/vscode#216490]
Our sandbox is using a multiroot workspace and a workspace file definition. The windows jump list is showing me 3-4 identical entries because it's always the same json mutliroot def file for different branches. I have to use the tooltip to know which one is which. 
Would be great if the Pinned Jump list would allow me to know which branch/folder I'm in without waiting for a tooltip.

![image](https://github.com/microsoft/vscode/assets/1581639/af2f154f-cc64-469b-bab5-59158877bdd0)

**Comment by bpasero** [comment]
How is this always the same entry, is the workspace called the same?

**Comment by sleutho** [comment]
Let's assume I'm working on the Manhatten project in two different sandboxes:

- Manhatten1/subfolder1/subfolder2/vscode/manhatten.code-workspace
- Manhatten2/subfolder1/subfolder2/vscode/manhatten.code-workspace

I'm working with a multiroot workspace configuration file, and that file is NOT under root. It's deeper inside my sandbox. 
The multiple folders above my code-workspace config file are all the same across multiple sandboxes. 

Not is my multi-root config file always called the same ( manhatten.code-workspace ). Multiple parent folders above it are as well ( subfolder1/subfolder2 ).
Would be nice to easily see Manhatten1 or Manhatten2. Right now only tooltips save the day.

**Comment by bpasero** [comment]
Yeah, today the name of a workspace is its file name, the parent folders are not taken into account.
